packages/lexical-editor/src/nodes/types.ts

```diff
@@ -0,0 +1,18 @@
+export type ThemeStyleType = "typography" | "colors" | "fonts";
+
+export interface ThemeStyleValue {
+    styleId: string;
+    type: ThemeStyleType;
+}
+
+export interface TypographyStylesNode {
+    setTypography: (typographyStyleId: string) => void;
+    getTypographyStyleId: () => string | undefined;
+    clearTypographyStyle: () => void;
+    hasTypographyStyle: () => boolean;
+}
+
+export interface TextNodeThemeStyles {
+    getThemeStyles: () => ThemeStyleValue[];
+    setThemeStyles: (styles: ThemeStyleValue[]) => void;
+}
```

Comment:
```suggestion
export interface TextNodeThemeStyles {
    getThemeStyles: () => ThemeStyleValue[];
    setThemeStyles: (styles: ThemeStyleValue[]) => void;
}

export interface TypographyStylesNode {
    setTypography: (typographyStyleId: string) => void;
    getTypographyStyleId: () => string | undefined;
    clearTypographyStyle: () => void;
    hasTypographyStyle: () => boolean;
}
```

Let's do a minor reorder.